src/app/controllers/charting.controller.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * ngInject
+ */
+export function ChartingController($scope, $attrs, $element, Utils) {
+  let defaultMargins = {left: 30, top: 10, bottom: 30, right: 10};
+  $scope.config = {};
```

Comment:
Gotcha. Yeah the MOS repo was earlier on in our understanding of angular, and doesn't quite follow best practices as we know them now. If you have time to refactor a bit to cleanup the use of $scope, go for it, otherwise it's fine as is, maybe leave a TODO comment so folks coming back to this know its not ideal?